### PR TITLE
[RW-4548] Merge storage enums classes

### PR DESCRIPTION
**Pull request by als364**
In #3211 the defunct `common-api` directory was merged into the regular api directory. The `CommonStorageEnums` module from `common-api` was similar in function to the `DbStorageEnums` module, so I merged them so we wouldn't have two. Just copied all the functions from `CommonStorageEnums` into `DbStorageEnums`.

### CODE REVIEW

File: api/src/test/java/org/pmiops/workbench/db/model/StorageEnumsTest.java [hunk 1]
```diff
@@ -75,16 +75,16 @@ private Object storageToEnum(Short shortValue, Class enumClass)
   }
 
   /**
-   * Retrieve all Enum classes in DbStorageEnums and CommonStorageEnums
+   * Retrieve all Enum classes in DbStorageEnums and DbStorageEnums
```
Comment (ericsong) [inline api/src/test/java/org/pmiops/workbench/db/model/StorageEnumsTest.java]: Nit: this comment should be revised to just `DbStorageEnums`

Comment (als364) [reply api/src/test/java/org/pmiops/workbench/db/model/StorageEnumsTest.java]: Looooooool. I did a global stringy search/replace. This is what I get. Thank you!

Comment (jmthibault79) [reply api/src/test/java/org/pmiops/workbench/db/model/StorageEnumsTest.java]: Lines 23 and 87 have similar dupes

Comment (jmthibault79) [reply api/src/test/java/org/pmiops/workbench/db/model/StorageEnumsTest.java]: And here's a dumb little PR to fix https://github.com/all-of-us/workbench/pull/3259

**Pull request merged**